scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java

```diff
@@ -1304,53 +1419,53 @@ public String getTaskServerLogsByTag(String jobId, String taskTag)
     public void putThirdPartyCredential(String key, String value)
             throws NotConnectedException, PermissionException, KeyException {
         UserIdentificationImpl ident = frontendState.checkPermission("putThirdPartyCredential",
-                                                                     YOU_DO_NOT_HAVE_PERMISSION_TO_PUT_THIRD_PARTY_CREDENTIALS_IN_THE_SCHEDULER);
+                YOU_DO_NOT_HAVE_PERMISSION_TO_PUT_THIRD_PARTY_CREDENTIALS_IN_THE_SCHEDULER);
 
         HybridEncryptionUtil.HybridEncryptedData encryptedData = HybridEncryptionUtil.encryptString(value,
-                                                                                                    corePublicKey);
+                corePublicKey);
         dbManager.putThirdPartyCredential(ident.getUsername(), key, encryptedData);
     }
 
     @Override
     public Set<String> thirdPartyCredentialsKeySet() throws NotConnectedException, PermissionException {
         UserIdentificationImpl ident = frontendState.checkPermission("thirdPartyCredentialsKeySet",
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)